phoenix-protocol/src/main/MetaDataService.proto

```diff
@@ -87,9 +87,6 @@ message GetTableRequest {
   required int64 tableTimestamp = 4;
   required int64 clientTimestamp = 5;
   optional int32 clientVersion = 6;	
-  optional bool skipAddingParentColumns = 7;
-  optional bool skipAddingIndexes = 8;
-  optional PTable lockedAncestorTable = 9;
 }
```

Comment:
No long need any of these since we aren't combining columns on the server side anymore.